ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact-3-nodes/cucushift-installer-rehearse-gcp-ipi-compact-3-nodes-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-compact-3-nodes
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-provision
```

Comment:
@jinyunma when you have time, can you help update azure-ipi-compact workflow similarly, seem like we can this is simpler than introducing provision chain.

Reply:
ah, okay, will update on azure platform.